sktime/networks/base.py

```diff
@@ -25,3 +28,154 @@ def build_network(self, input_shape, **kwargs):
         output_layer : a keras layer
         """
         ...
+
+
+class BaseDeepNetworkPyTorch(BaseForecaster, ABC):
+    """Abstract base class for deep learning networks using torch.nn."""
+
+    _tags = {
+        "python_dependencies": "torch",
+    }
+
+    def __init__(self):
+        super().__init__()
```

Comment:
Can you add some default values for all attributes used by this class and defined only in subclasses? This would enable the intellisense to work properly.